packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistory.tsx

```diff
@@ -0,0 +1,119 @@
+import { useMemo } from 'react'
+import { Tab } from '@headlessui/react'
+import { twMerge } from 'tailwind-merge'
+import { useAccount } from 'wagmi'
+
+import { useCompleteMultiChainTransactions } from '../../hooks/useCompleteMultiChainTransactions'
+import { TransactionHistoryTable } from './TransactionHistoryTable'
+import {
+  isTxClaimable,
+  isTxCompleted,
+  isTxExpired,
+  isTxFailed,
+  isTxPending
+} from './helpers'
+import { MergedTransaction } from '../../state/app/state'
+import { TabButton } from '../common/Tab'
+
+const roundedTabClasses =
+  'roundedTab ui-not-selected:arb-hover relative flex flex-row flex-nowrap items-center gap-0.5 md:gap-2 rounded-tl-lg rounded-tr-lg px-2 md:px-4 py-2 text-base ui-selected:bg-white ui-not-selected:text-white justify-center md:justify-start grow md:grow-0'
+
+export const TransactionHistory = () => {
+  const { address } = useAccount()
+
+  const {
+    data: { transactions },
+    loading,
```

Comment:
For the first few seconds after logging in, the `loading` flag doesn't seem to work well. It shows No pending transactions, but it shows transactions after some seconds. There is no loader as well.
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/4e1e4fa1-2be2-47f8-a7e4-803122748142)
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/aaa25fa7-ee01-4f22-9c39-4891d0ec527d)

Reply:
fixed